aiohttp/web_reqrep.py

```diff
@@ -321,6 +321,15 @@ def json(self, *, loads=json.loads, loader=None):
         return loads(body)
 
     @asyncio.coroutine
+    def multipart(self, *, reader=multipart.MultipartReader):
+        """Return async iterator to process BODY as multipart."""
+        if not self.content_type.startswith('multipart/'):
+            warnings.warn('Attempt to process multipart request with'
```

Comment:
I mean if `Content-Type` is not started with `multipart/` it's maybe not a warning but an exception?